packages/api/src/hooks/useTradingPlatformInvestorPasswordChange.ts

```diff
@@ -0,0 +1,22 @@
+import { useCallback } from 'react';
+import useRequest from '../useRequest';
+
+type TPayload = Parameters<
+    ReturnType<typeof useRequest<'trading_platform_investor_password_change'>>['mutate']
+>[0]['payload'];
+
+/** A custom hook that change the Trading Platform Investor Password. */
+const useTradingPlatformInvestorPasswordChange = () => {
```

Comment:
@hamza-deriv Could we name it something better? No need to follow API naming, We are trying to make entity-based hooks.

I don't know what is `Trading Platform Investor`, Definitely a new joiner also doesn't know, So would be great if we could name it something more meaningful 🙏🏻 

`useChangeCFDAccountPassword`? 🤔

Reply:
Investor Password is ony for MT5 where the MT5 Account manager are termed as Investor which manages your account, and the password is specifically for this purpose if we change it to `useChangeCFDAccountPassword` then there is no difference between a simple password and Investor Password, and we don't have password change for MT5, Deriv X, its only reset can which can be done using `useVerifyEmail` hook.